### DISCUSSION TITLE
Nested modules into a single file

### DISCUSSION THREAD

**Discussion by jnoverwolf** [tgreyuk/typedoc-plugin-markdown#730] [Help]
Hi I have a TS file with multiple `namespace` declarations. 
When I run the plugin, using the `flattenOutputFiles=true`  I still get multiple files for each declared namespace (`<nanespace_parent>.Namepscae.<namespace_name>`). I would like to have these files all be in the same entry point file, so that I have one file that shows all the namespaces.
Is this possible?

**Comment by tgreyuk** [comment]
Hello,

Its not currently possible. Each module and namespace are always published to a separate file currently as otherwise there could be limitless nesting of modules and namespaces. 

Having said that we could implement something that lists each module and namespace on a single document under multiple h1 headings (if that makes sense).

**Reply by jnoverwolf** [reply]
Yes that makes sense. However, would a custom tag be able to aggregate the namespaces for me?
